prusti-tests/tests/verify_overflow/fail/ghost/sequences.rs

```diff
@@ -0,0 +1,18 @@
+#[macro_use]
```

Comment:
Is this required? I thought we can already use e.g. `predicate!` without the attribute.

Reply:
I am currently confused about the declarative macros, no amount of attributes seems to make them work.
I always get ```error: cannot find macro `ghost` in this scope```